src/test/java/org/folio/rest/utils/ResourceClients.java

```diff
@@ -36,6 +36,10 @@ public static ResourceClient accountsTransferClient(String accountId) {
     return buildUrlForAction(accountId, "transfer");
   }
 
+  public static ResourceClient accountsCancelClient(String accountId) {
```

Comment:
Let's rename this method to buildAccountCancelClient  (also, rename others similarly)

Reply:
renamed